backend/environment/env.social-app-backend.docker.template

```diff
@@ -4,6 +4,9 @@
 # Base URL for the account-service
 ACCOUNT_SERVICE_URL=http://account-service-api:3000
 
+# Base URL for the content-publishing-service
+CONTENT_PUBLISHER_URL=http://content-publishing-service-api:3000
```

Comment:
Side note:

I know these are in here for the docker compose, but I'm curious why not just set these values in the compose file where the hostnames are also setup?

Reply:
I want to have parity between docker/non-docker environment setups, and not have some env vars defined here and some in the docker-compose file.